config.yml

```diff
@@ -1,6 +1,9 @@
 RunConfig:
   mode: LICHESS # running mode, choose from: LICHESS, UCI
 
+EvaluatorConfig:
```

Comment:
I wonder if it's best to put this under `SearcherConfig`, since evaluator is a subset of searcher. I guess you'll need `evaluator_config` (see `move_order_config` on line 14) instead given how we've implemented from_dict (inconsistent I know),